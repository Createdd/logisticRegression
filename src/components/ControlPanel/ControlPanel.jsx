import React from "react";
import PropTypes from "prop-types";

const ControlPanel = props => (
  <div>
    <div className="row">
      <a
        className="waves-effect waves-light btn teal lighten-1"
        onClick={() => props.switchAge("19")}
      >
        Up to 19 year olds
      </a>
      <a
        className="waves-effect waves-light btn teal lighten-1"
        onClick={() => props.switchAge("70+")}
      >
        From 70+ year olds
      </a>
      <a
        className="waves-effect waves-light btn teal lighten-1"
        onClick={() => props.switchAge("")}
      >
        All Ages
      </a>
    </div>
    <div className="row">
      <a
        className="waves-effect waves-light btn red lighten-1"
        onClick={props.calcRegression}
      >
        Toggle Regression Curve
      </a>
    </div>
  </div>
);

export default ControlPanel;

ControlPanel.PropTypes = {
  switchAge: PropTypes.func
};
