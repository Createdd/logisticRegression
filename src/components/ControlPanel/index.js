export { default } from "./ControlPanel.jsx";

