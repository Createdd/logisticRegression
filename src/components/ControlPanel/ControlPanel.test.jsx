import * as React from "react";
import * as renderer from "react-test-renderer";

import ControlPanel from "./ControlPanel";

describe("Test ControlPanel", () => {
  it("renders snapshot", () => {
    const tree = renderer
      .create(<ControlPanel switchAge={() => {}} calcRegression={() => {}} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
