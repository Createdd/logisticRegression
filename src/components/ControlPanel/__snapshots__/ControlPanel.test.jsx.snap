// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test ControlPanel renders snapshot 1`] = `
<div>
  <div
    className="row"
  >
    <a
      className="waves-effect waves-light btn teal lighten-1"
      onClick={[Function]}
    >
      Up to 19 year olds
    </a>
    <a
      className="waves-effect waves-light btn teal lighten-1"
      onClick={[Function]}
    >
      From 70+ year olds
    </a>
    <a
      className="waves-effect waves-light btn teal lighten-1"
      onClick={[Function]}
    >
      All Ages
    </a>
  </div>
  <div
    className="row"
  >
    <a
      className="waves-effect waves-light btn red lighten-1"
      onClick={[Function]}
    >
      Toggle Regression Curve
    </a>
  </div>
</div>
`;
